examples/test-app/steps/CheckYourAnswers.step.js

```diff
@@ -2,14 +2,33 @@ const {
   CheckYourAnswers: CYA,
   section
 } = require('@hmcts/one-per-page/checkYourAnswers');
+const { goTo, action } = require('@hmcts/one-per-page/flow');
+const request = require('request-promise-native');
 
 class CheckYourAnswers extends CYA {
+  constructor(...args) {
+    super(...args);
+    this.sendToAPI = this.sendToAPI.bind(this);
```

Comment:
do you need to bind this?

Reply:
yep, `sendToAPI` calls `this.journey`. If it isn't bound then the instance won't be `this`. ES6 is infuriating sometimes.